### ISSUE TITLE
Where are the F# communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/fsharp#550]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is F# supported by one or more large organizations?
* Does F# have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about F#? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for F#? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in F#? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in F#?
* Are there popular newsletters for F#?
* Is F# taught at programming bootcamps? (If so, what are some examples?)
* Is F# taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about F#?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by ErikSchierboom** [comment]
> Is F# supported by one or more large organizations?

Yes, it is supported by Microsoft. We used to have Xamarin as a medium-to-large organization supporting C#, but they have been acquisitioned by Microsoft.

> Does F# have an official community manager?

Yes, [Phillip Carter](https://github.com/cartermp).

> Do you know of specific communities (online or offline) that are enthusiastic about F#? (Chat communities, forums, meetups, student clubs, etc)

There is the [F# Foundation](http://foundation.fsharp.org/), which has a [Slack page](https://fsharp.org/guides/slack/). Also, the F# channel on the [functional-programming Slack page](https://functionalprogramming.slack.com/messages/C045LHLTH/details/) is also quite active.

> Are there popular conferences for F#? (If so, what are some examples?)

There is [.NET Fringe](http://dotnetfringe.org/), [fsharpConf](http://fsharpconf.com/), [FableConf](http://fable.io/fableconf/#home), [Open F#](https://www.openfsharp.org/), and probably more.

> Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in F#? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

I don't know.

> Are there specific groups or programs dedicated to mentoring people in F#?

Yes, through the [F# mentorship program](https://fsharp.org/mentorship/about.html).

> Are there popular newsletters for F#?

Yes, the [F# weekly](https://sergeytihon.com/category/f-weekly/) newsletter.

> Is F# taught at programming bootcamps? (If so, what are some examples?)

I don't know.

> Is F# taught at universities? (If so, what are some examples?)

I don't know.

**Comment by kytrinyx** [comment]
Lovely, thank you! One of the things I want to do once the v2 site is public is reach out to the fantastic folks that are running both the F# and C# community efforts at Microsoft to see how we can make Exercism a better resource for them.

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. (Should you come across any other good sources, feel free to add them here, I'll be coming back to this). ❤️ 💛 💚 💙 💜 🖤 Thanks!
